en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md

```
@@ -17,67 +17,70 @@ security researcher community relationship.
+| :---------------------------------------------------------------------------------------------------------- |
+| [Alex Williams from Pellera Technologies](https://www.linkedin.com/in/alexanderwilliamscyber)               |
+| [Claire Wang](https://clairewang.net/)                                                                      |
+| [Lasantha Karunarathne](https://www.linkedin.com/in/lasakaru/)                                              |
+| [Maneesha Dewmina](https://www.linkedin.com/in/maneesha-dewmina/)                                           |
+| [Manisha Dilshan](https://www.linkedin.com/in/manisha-dilshan-b08ba3215)                                    |
+| [Michał Majchrowicz, Marcin Wyczechowski, and Paweł Zdunek — members of the AFINE Team](https://afine.com/) |
+| **Omri Inbar**                                                                                              |
+| [Robert C. Raducioiu](https://it.linkedin.com/in/rbct)                                                      |
+| [Suraj Theekshana and Ishan Nim (株式会社CyberCrew)](https://cyber.spool.co.jp/)                             |
+| [Thinh Dang](https://www.linkedin.com/in/thinh-dang-bb4b72170/)                                             |
+| [Maneesha Dewmine]()                                                                                        |
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)